src/views/Alteration.vue

```diff
@@ -58,6 +59,11 @@ export default class Alteration extends Mixins(LegalApiMixin, FilingTemplateMixi
 
   /** Called when this component is mounted. */
   private mounted (): void {
+    if (!featureFlags.getFlag('alteration-ui-enabled')) {
+      alert('Alterations are under contruction. Please check again later.')
+      return
```

Comment:
Does `return` within a `mounted` method prevent the view from mounting?

Reply:
No, but it means code below this doesn't get executed.

There's a `beforeMount` event that we could perhaps use if needed. For now I think this is OK.
https://vuejs.org/v2/guide/instance.html#Lifecycle-Diagram

Reply:
Sounds good!